llama_hub/file/pymu_pdf/base.py

```diff
@@ -9,6 +9,12 @@
 class PyMuPDFReader(BaseReader):
     """Read PDF files using PyMuPDF library."""
 
+    def load_data(
+        self, file_path: Union[Path, str], extra_info: Optional[Dict] = None
+    ) -> List[Document]:
+        """Loads list of documents from PDF file and also accepts extra information in dict format."""
+        return self.load(file_path, metadata=True, extra_info=extra_info)
+
     def load(
```

Comment:
I wonder if we should just remove the load function -- I feel like it technically shouldn't even be there 😅

Reply:
I just kept it for backwards compatibility